R/pkg/R/generics.R

```diff
@@ -152,9 +146,9 @@ setGeneric("getNumPartitions", function(x) { standardGeneric("getNumPartitions")
 # @export
 setGeneric("numPartitions", function(x) { standardGeneric("numPartitions") })
 
-# @rdname persist
-# @export
-setGeneric("persist", function(x, newLevel) { standardGeneric("persist") })
+setGeneric("partitionByRDD", function(x, ...) { standardGeneric("partitionByRDD") })
```

Comment:
Ah yes - you are right. We do have a `partitionBy` in windowSpec. Renaming the RDD one is fine -- we might need to move the `partitionBy` generic to the right place in `generics.R` ?